### DISCUSSION TITLE
Translatable settings fields

### DISCUSSION THREAD

**Discussion by panakour** [spatie/laravel-settings#115] [Ideas]
Is it possible to use [spatie translatable](https://github.com/spatie/laravel-translatable) to make the settings translatable ?

**Comment by rubenvanassche** [comment]
Hi @panakour,

That's not possible and a bit out of scope concerning this package. You could create a DTO that converts to an array and vice versa and store this as a setting

**Reply by panakour** [reply]
Hi @rubenvanassche,

Thank you for letting me know.
